src/library_syscall.js

```diff
@@ -660,7 +666,9 @@ var SyscallsLibrary = {
   __syscall140: function(which, varargs) { // llseek
     var stream = SYSCALLS.getStreamFromFD(), offset_high = SYSCALLS.get(), offset_low = SYSCALLS.get(), result = SYSCALLS.get(), whence = SYSCALLS.get();
     var offset = offset_low;
+#if WASM_BACKEND == 0
```

Comment:
Because https://github.com/kripken/emscripten/blob/incoming/system/lib/libc/musl/src/unistd/lseek.c calls this with an `off_t` shifted right by 32, to split what it assumes is a 64bit value into high and low bits. Because asmjs doesn't handle 64bit integers, we define `off_t` as just an `int` (https://github.com/kripken/emscripten/pull/4813/commits/3aa7fde0c1512db0315b0120faad47ec6911e610#diff-f5c20f80e42860c7a8b52346283d5048R226), making the rightshift undefined behavior.
I don't remember what specifically about the wasm backend makes the value be garbage here (because it gets the same definition of `off_t` as an `int`), but it tripped this assert a bunch. Now I thought I remembered `lseek.c` changed in this PR to introduce the undefined shift, but it didn't so I'm not entirely sure why this assert gets hit *now* when it didn't previously.